install-global.sh

```
@@ -373,6 +412,94 @@ install_cursor_commands() {
+        "python.mdc"
+        "perl.mdc"
+        "database.mdc"
+    )
+
+    for mdc in "${mdc_files[@]}"; do
+        local basename="${mdc%.mdc}"
+        local source_url="$REPO_URL/.cursor/rules/$mdc"
+        local target_md="$basename.md"
+
+        if [[ "$PLAN_MODE" == true ]]; then
+            local tmp_mdc=$(mktemp)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)